cccatalog-api/cccatalog/api/controllers/search_controller.py

```diff
@@ -378,27 +378,27 @@ def related_images(uuid, index, request, filter_dead):
     return results, result_count
 
 
-def get_providers(index):
```

Comment:
Are we also exposing providers in the API? because we should

Reply:
`/v1/sources` is a wrapper around get_sources, formerly get_provider. That will continue to be exposed.

If we need to show `providers` in the new sense through an endpoint, let's make a separate ticket specifying what we need that endpoint for and how it will be used by the frontend.

I would rather not rush that change into this PR / duplicate the existing design of the `/v1/sources` endpoint for `/v1/providers` because there are problems with counting a growing set of half a billion images in realtime.

Reply:
A separate ticket makes sense, thanks.